src/core/common_path.py

```diff
@@ -0,0 +1,6 @@
+from pathlib import Path
+
+
+class CommonPaths:
```

Comment:
@vasylboliuk please add a docstring describing what this class is used for. This is required by the PEP8

Reply:
Done